src/lib/mongoBackend/compoundValueBson.h

```diff
@@ -37,14 +37,14 @@
 *
 * compoundValueBson (for objects) -
 */
-extern void compoundValueBson(std::vector<orion::CompoundValueNode*> children, mongo::BSONObjBuilder& b);
+extern void compoundValueBson(const std::vector<orion::CompoundValueNode*>& children, mongo::BSONObjBuilder& b);
```

Comment:
> However it is not a bug fix, only a fix to save space on the stack.

That's why I say to use "Hardening:" and not "Fix:" in the CNR line ;)

"Low level will" be going change by change mentioning every changed function in CNR. Adding a general comment like the one I proposed doesn't hurt.